python/ray/rllib/example.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""
+This demonstrates the RLlib algorithm API through a simple bakeoff.
+"""
```

Comment:
Let's stick with Google style docstrings. I've been using http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for reference.